src/main/java/games/strategy/triplea/ui/AbstractMovePanel.java

```diff
@@ -244,7 +245,7 @@ protected final void display(final PlayerID id, final String actionLabel) {
       this.actionLabel.setText(id.getName() + actionLabel);
       add(leftBox(this.actionLabel));
       if (setCancelButton()) {
-        add(leftBox(new JButton(SwingComponents.newAbstractAction(this::cancelMove))));
+        add(leftBox(new JButton(cancelMoveAction)));
```

Comment:
There is no redundancy because we never explicitly set the cancel button text.  The label is only passed to `newAbstractAction()`, and, as described above, the `JButton` constructor uses the action name to initialize the button text.

I will investigate changing the `Action` field to a corresponding `JButton`.  I think that will work.